impl/c-qube/src/utils/logger.ts

```diff
@@ -0,0 +1,31 @@
+const winston = require('winston');
+const fs = require('fs');
+
+function setupLogger(logFileName: string, logLevel: string = 'info') {
+  const logDir = 'logs';
+
+  if (!fs.existsSync(logDir)) {
+    fs.mkdirSync(logDir);
+  }
+
+  const logger = winston.createLogger({
```

Comment:
Can you make the functionality for adding log levels available as function calls on the logger object? Similar to how it happens in the native nest js logger. 
Something like:
```ts
Logger.log('....');
Logger.error('...');
Logger.warn('...');
Logger.verbose('...');
```